jwst/wfs_combine/wfs_combine.py

```diff
@@ -591,54 +643,16 @@ def get_overlap(sci_int_1, sci_int_2, nom_off_x, nom_off_y):
 
     # From the nominal offsets, determine array indices to shift image #2
     #     onto frame #1
-    ix, fx = get_index_range(nom_off_x, sci_int_2.shape[1])
-    iy, fy = get_index_range(nom_off_y, sci_int_2.shape[0])
+    initial_x_1, final_x_1, initial_x_2, final_x_2 = get_index_range(nom_off_x, sci_int_2.shape[1])
+    initial_y_1, final_y_1, initial_y_2, final_y_2 = get_index_range(nom_off_y, sci_int_2.shape[0])
 
-    sub_1 = sci_int_1[:fy - iy, :fx - ix]
-    sub_2 = sci_int_2[iy:fy, ix:fx]
+    sub_1 = sci_int_1[initial_y_1:final_y_1, initial_x_1:final_x_1]
+    sub_2 = sci_int_2[initial_y_2:final_y_2, initial_x_2:final_x_2]
 
     return sub_1, sub_2
 
 
-def optimize_offs(sci_nai_1, sci_nai_2):
-    """
-    Short Summary
-    -------------
-    Calculate refined values for the offsets. Find the nearby integer
-    relative offsets between the nominally aligned interpolated images
-    which minimizes the square of the difference between the 2 images.
-
-    Parameters
-    ----------
-    sci_nai_1: 2d float array
-        nominally aligned, interpolated SCI array for image 1
-    sci_nai_2: 2d float array
-        nominally aligned, interpolated SCI array for image 2
-
-    Returns
-    -------
-    max_off_x: integer
-        offset in x-direction which minimizes array difference
-    max_off_y: integer
-        offset in y-direction which minimizes array difference
-
-    """
-
-    max_diff = 0.
-
-    for off_y in range(-OFF_DELTA, OFF_DELTA + 1):
-        for off_x in range(-OFF_DELTA, OFF_DELTA + 1):
-            this_diff = calc_cor_coef(sci_nai_1, sci_nai_2, off_x, off_y)
-
-            if (this_diff > max_diff):
-                max_diff = this_diff
-                max_off_x = off_x
-                max_off_y = off_y
-
-    return max_off_x, max_off_y
-
-
-def calc_cor_coef(sci_nai_1, sci_nai_2, off_x, off_y):
+def calc_refined_offsets(sci_nai_1, sci_nai_2, off_x, off_y, psf_size):
```

Comment:
I second Howard's question. Also, a description of this new 'refined' method should be documented in the epydocs and in readthedocs.

Reply:
I've updated the epydocs and the readthedocs to match the current version.